app/src/main/java/ch/epfl/sdp/LeaderboardActivity.java

```diff
@@ -0,0 +1,86 @@
+package ch.epfl.sdp;
+
+import android.app.Activity;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LeaderboardActivity extends AppCompatActivity {
+
```

Comment:
Those attributes are also in the LeaderboardAdapter right? Do we need those?

Reply:
Yeah, we need this, this how recyclerview works.

Reply:
Ok :)